core/kernel/ldelf_syscalls.c

```diff
@@ -304,6 +305,8 @@ TEE_Result ldelf_syscall_map_bin(vaddr_t *va, size_t num_bytes,
 		prot |= TEE_MATTR_UW | TEE_MATTR_PW;
 	if (flags & LDELF_MAP_FLAG_EXECUTABLE)
 		prot |= TEE_MATTR_UX;
+	if (flags & LDELF_MAP_FLAG_BTI)
+		prot |= TEE_MATTR_GAURDED;
```

Comment:
Shouldn't this flag contain "`BTI`"? I find the term "guarded" a bit too vague.

Reply:
The term guarded is used in the Arm ARM.

Reply:
Fine then :)